### DISCUSSION TITLE
Q about limitations

### DISCUSSION THREAD

**Discussion by yaroslav-hub** [rabbitmq/rabbitmq-delayed-message-exchange#274] [Q&A] [answered]
Hi!
I'm interested in information about limitations of delayed exchanges in latest version of plugin.
When I did a research, I found this article: [TAP](https://www.cloudamqp.com/blog/how-to-avoid-the-pitfalls-of-the-delayed-message-exchange.html). And I want to know if there are any changes with this aspects. Could you describe all of points from this article if it changed.
Thanks!

**Comment by mkuratczyk** [comment, marked as answer]
All the key limitations still apply and are mentioned in the README. This plugin is effectively approaching its end of life.
